Sources/Plasma/PubUtilLib/plScene/plSceneNode.cpp

```diff
@@ -94,7 +94,7 @@ void plSceneNode::Read(hsStream* s, hsResMgr* mgr)
     hsKeyedObject::Read(s, mgr);
 
     uint32_t n;
-    int i;
+    uint32_t i;
```

Comment:
Could be made a local variable in each `for` loop (and also `n` could be merged with its first assignment).